dependencies/lib-fltk/builder/CMakeLists.txt.in

```diff
@@ -33,7 +33,7 @@ externalproject_add(fltk
         -DOPTION_USE_XFT=OFF
         -DOPTION_USE_XRENDER=OFF
         -DOPTION_USE_XDBE=OFF
-        -DOPTION_BUILD_EXAMPLES=OFF
```

Comment:
This flag got removed upstream?

Reply:
It got obsolete, I'm getting this when reverting to the previous version.

```
CMake Warning at CMake/options.cmake:113 (message):
  'OPTION_BUILD_EXAMPLES' is obsolete, please use 'FLTK_BUILD_TEST' instead.
Call Stack (most recent call first):
  CMakeLists.txt:79 (include)
```